app/helpers/application_helper.rb

```diff
@@ -30,4 +30,23 @@ def contractor_name(contractor)
   def current_contractors_project?(project)
     project.contractor == current_contractor
   end
+
+  def project_actions(status)
+    case status
+    when "Pending"
+      concat( link_to "Submit Payment", contractor_project_payment_path(@project), id: "checkout-button")
```

Comment:
Line is too long. [105/80]<br>Add parentheses to nested method call `link_to "Submit Payment", contractor_project_payment_path(@project), id: "checkout-button"`.<br>Space inside parentheses detected.